selftests/functional/plugin/test_jsonresult.py

```diff
@@ -0,0 +1,21 @@
+import json
+from os import path
+
+from avocado.utils import process
+from selftests.utils import AVOCADO, TestCaseTmpDir
+
+
+class JsonResultTest(TestCaseTmpDir):
+
```

Comment:
Was this added accidentally?

Reply:
No, it is the test for this feature, so we will be able to catch the #4750 in the future. Unfortunately, it is in conflict with #4753, because those tests didn't exist, but I will resolve that after one of those will be merged.

Reply:
Ok, I saw you were adding the same file in both pull requests and it was confusing. Please, add a comment in the git log as well.

Reply:
ok, no problem